metpy/calc/thermo.py

```diff
@@ -734,3 +734,107 @@ def psychrometric_vapor_pressure_wet(dry_bulb_temperature, wet_bulb_temperature,
     """
     return (saturation_vapor_pressure(wet_bulb_temperature) - psychrometer_coefficient *
             pressure * (dry_bulb_temperature - wet_bulb_temperature).to('kelvin'))
+
+
+@exporter.export
+@check_units('[dimensionless]', '[temperature]', '[pressure]')
+def relative_humidity_from_mixing_ratio(mixing_ratio, temperature, pressure):
+    r"""Calculate the relative humidity from mixing ratio, temperature, and pressure.
+
+    Parameters
+    ----------
+    mixing_ratio: `pint.Quantity`
+        Dimensionless mass mixing ratio
+    temperature: `pint.Quantity`
+        Air temperature
+    pressure: `pint.Quantity`
+        Total atmospheric pressure
+
+    Returns
+    -------
+    `pint.Quantity`
+        Relative humidity
+
+    Notes
+    -----
+    Formula from [Hobbs 1977]_ pg. 74.
+    .. math:: RH = 100 \frac{w}{w_s}
+
+    * :math:`RH` is relative humidity
+    * :math:`w` is mxing ratio
+    * :math:`w_s` is the saturation mixing ratio
+
+    See Also
+    --------
+    saturation_mixing_ratio
+
+    """
+    return (100 * units.percent *
+            mixing_ratio / saturation_mixing_ratio(pressure, temperature))
+
+
+@exporter.export
+@check_units('[dimensionless]')
+def mixing_ratio_from_specific_humidity(specific_humidity):
+    r"""Calculate the mixing ratio from specific humidity. Formula from [Salby1996]_ pg. 118.
```

Comment:
The mention of the formula should be a separate paragraph. The first line of the docstring should just be a single sentence by itself.